File: CJY/BJ/No_11720.js
const rl = require("readline").createInterface({input: process.stdin, output: process.stdout});
let idx = 0;

rl.on("line", (s) => {
    if(idx) {
        const arr = s.split("");
        let n = 0;
        for (let i = arr.length; i--;) n += parseInt(arr[i]);
        console.log(n);
        rl.close();
    }
    idx++;
}).on("close", () => {process.exit();});

File: CJY/BJ/No_11720.java
import java.io.*;

public class No_11720 {
    public static void main(String[] args) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        br.readLine();
        String[] arr = br.readLine().split("");
        br.close();

        BufferedWriter bw = new BufferedWriter(new OutputStreamWriter(System.out));
        int sum = 0;
        for(String s : arr) sum += Integer.parseInt(s);
        bw.write(String.valueOf(sum));
        bw.flush();
        bw.close();
    }
}